README.md

```diff
@@ -50,7 +50,7 @@ To create a new project, run:
 This will prompt for parameters for project initialization. Some of these parameters are required to get started:
  * ``project_name``: name of the current project
  * ``cloud``: Cloud provider you use with Databricks (AWS, Azure, or GCP)
- * ``cicd_platform`` : CI/CD platform of choice (GitHub Actions or Azure DevOps)
+ * ``cicd_platform`` : CI/CD platform of choice (GitHub Actions or Github Enterprise Actions or Azure DevOps)
```

Comment:
How about (Github Actions, Github Actions for enterprises or Azure DevOps)? 
The "cicd_platform" variable needs to be changed accordingly.

I wasn't able to find term "Github Enterprise Actions".
Github doc
https://docs.github.com/en/enterprise-server@3.8/admin/github-actions/getting-started-with-github-actions-for-your-enterprise/about-github-actions-for-enterprises

Reply:
I wouldn't do Github Enterprise Actions... "Github Actions (Enterprise)" or "Github Actions for Enterprises" maybe? The latter seems to be the official name.

Reply:
Looks like the official name is [GitHub Actions for GitHub Enterprise Server](https://docs.github.com/en/enterprise-server@3.5/admin/github-actions/getting-started-with-github-actions-for-your-enterprise/getting-started-with-github-actions-for-github-enterprise-server)

Reply:
Damn, that's a mouthful but it's better (can't argue with official).